quarkus-opentelemetry-exporter-gcp/deployment/src/test/java/io/quarkiverse/opentelemetry/exporter/gcp/deployment/GcpExporterDisabledTest.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkiverse.opentelemetry.exporter.gcp.deployment;
+
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.quarkiverse.opentelemetry.exporter.gcp.runtime.LateBoundBatchSpanProcessor;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class GcpExporterDisabledTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .withEmptyApplication()
+            .overrideConfigKey("quarkus.opentelemetry.tracer.exporter.gcp.enabled", "false");
```

Comment:
We should also have a positive test, making sure the processor was activated.

Reply:
Test added. Please see my comments below.